subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/PerformanceTestScenarioDefinitionVerifier.groovy

```diff
@@ -29,7 +29,11 @@ class PerformanceTestScenarioDefinitionVerifier {
         File newFile = new File(args[1])
         PerformanceTestScenarioDefinition oldJson = new ObjectMapper().readValue(oldFile, PerformanceTestScenarioDefinition).sort()
         PerformanceTestScenarioDefinition newJson = new ObjectMapper().readValue(newFile, PerformanceTestScenarioDefinition).sort()
-        if (oldJson != newJson) {
+
+        oldJson.removeIgnoredScenarios(newJson.ignoredScenarios)
```

Comment:
That means that the splits containing the tests are still executed on CI, only that the test is ignored in the split. What happens if a split only contains one method which is ignored? Would the test task then fail because it didn't run any tests?